packages/devtools_app/integration_test/test_infra/run/_test_app_driver.dart

```diff
@@ -434,6 +434,7 @@ enum TestAppDevice {
     TestAppDevice.cli: [
       'debugger_panel_test.dart',
       'eval_and_browse_test.dart',
+      'eval_and_inspect_test.dart',
```

Comment:
is there a reason we are adding a new test instead of removing the exclude entry for `eval_and_browse_test.dart` on line 429? At first glance it looks like most of the code in `eval_and_inspect_test.dart` is copied from `eval_and_browse_test.dart`.

Reply:
They are testing similar functionality but are not exactly the same. I can move some common code into a shared file  after I figure out why the new test is always passing... 

- eval_and_browse_test tests eval-and-browse functionality on the memory page (the page is not currently supported for web, so it is not enabled still to run  for web platform)

- eval_and_inspect_test tests eval-and-browse feature for the inspector page, but it should work on other platforms as well, it is not really web-specific. This test also verifying  the main changes in the current PR (enabling eval-and-browse for web). I think it is good to add for other platforms as well.

Btw, I don't fully understand the testing scenarios yet - what is the cli app device and when is it running in the CI? How can I test it locally? I currently disabled the new test for it just to match what the eval_and_browse does.

Reply:
I am thinking about renaming the tests to `eval_and_browse_memory_test.dart`  and `eval_and_browse_inspector_test.dart`, and moving the common functionality into `eval_and_browse_shared.dart`